invenio/legacy/webbasket/templates.py

```diff
@@ -1777,14 +1777,14 @@ def tmpl_add(self,
         return out
 
     def tmpl_confirm_delete(self, bskid,
-                            (nb_users, nb_groups, nb_alerts),
+                            (nb_users, nb_groups, 0),
```

Comment:
can you change it to `nb` and later in function do `nb_users, nb_groups, dummy = nb`?